src/scripts/configure.ac

```diff
@@ -2,7 +2,7 @@
 # Increment the last 2 digits when making an API change
```

Comment:
Out of curiosity, is this comment (regarding the last 2 digits) still correct?

Reply:
I don't know if it's policy, but it makes sense. The "version number" and the "release" aren't necessarily the same thing.

Reply:
I think we should change it to be consistent with the version number lest a divergence cause confusion. I'll make a separate PR for changing the comment.

Reply:
Specifically, I think we should abide by the [Semantic Versioning](https://semver.org/) guidelines of:

> Given a version number MAJOR.MINOR.PATCH, increment the:
> 
>     MAJOR version when you make incompatible API changes,
>     MINOR version when you add functionality in a backwards compatible manner, and
>     PATCH version when you make backwards compatible bug fixes.

Reply:
Updated in #2740.